pom.xml

```diff
@@ -38,6 +38,12 @@
             <artifactId>azure-spring-boot-starter-active-directory</artifactId>
             <version>3.6.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
```

Comment:
Added dependency to allow the testing and overall program to run smoothly and reduce other areas of technical debt.
https://www.techopedia.com/definition/27913/technical-debt